kable-core/src/appleMain/kotlin/bluetooth/IsSupported.kt

```diff
@@ -0,0 +1,49 @@
+package com.juul.kable.bluetooth
+
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.channels.onFailure
+import kotlinx.coroutines.flow.callbackFlow
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import platform.CoreBluetooth.CBCentralManager
+import platform.CoreBluetooth.CBCentralManagerDelegateProtocol
+import platform.CoreBluetooth.CBCentralManagerOptionShowPowerAlertKey
+import platform.CoreBluetooth.CBManagerState
+import platform.CoreBluetooth.CBManagerStateResetting
+import platform.CoreBluetooth.CBManagerStateUnknown
+import platform.CoreBluetooth.CBManagerStateUnsupported
+import platform.darwin.NSObject
+
+// Prevent triggering permissions dialog.
+// https://chrismaddern.com/determine-whether-bluetooth-is-enabled-on-ios-passively/
+private val options = mapOf(CBCentralManagerOptionShowPowerAlertKey to false) as Map<Any?, *>?
```

Comment:
This actually works? Has this been tested? The article is 10 years old.

Reply:
Yup. I tested it, and it works. 🎉

Reply:
Ya, right?! It was a lucky find, and even luckier that it actually worked. 😆